src/configureStore.js

```diff
@@ -31,7 +31,7 @@ export default function configureStore(initialState, history) {
       errorReporter,
       middleware
     ),
-    __CLIENT__ ? persistState(['authenticated', 'idToken', 'user'], { key: 'ndla:sti', slicer }) : f => f,
+    __CLIENT__ ? persistState(['authenticated', 'accessToken', 'user', 'accessToken'], { key: 'ndla:sti', slicer }) : f => f,
```

Comment:
Her er accessToken listet to ganger?

Reply:
Ah. Good catch. Både `user` og den andre `accessToken` skal bort... Fikser...